src/parser/TTML.cpp

```diff
@@ -208,12 +208,9 @@ bool TTML2SRT::Parse(const void *buffer, size_t buffer_size, uint64_t timescale,
   XML_SetElementHandler(parser, start, end);
   XML_SetCharacterDataHandler(parser, text);
 
-  XML_Status retval = XML_Parse(parser, (const char*)buffer, buffer_size, done);
+  XML_Parse(parser, (const char*)buffer, buffer_size, done);
   XML_ParserFree(parser);
 
-  if (retval == XML_STATUS_ERROR)
```

Comment:
I don't understand this removal

Reply:
without this removal, (non-english) subtitles are rendered once they are parsed like this: http://imgur.com/a/wFkcO

I could not find the root cause, but removing the return fixes the issue

here is a [bulgarian](https://test.net.in/Fragments(textstream_BUL_CAPT=14400000000)) fragment sample (does not match the screenshot above but you get the idea) that broke

another [english](https://test.net.in/Fragments(textstream_eng_SUBT=14400000000)) sample that worked

Reply:
just to add, as a workaround, we adapted our addon to download external .dfxp subtitle file, convert it to srt and and tell the player to use load external subs. so this is not high prio. would be nice if you have an idea what's the root cause :)

Reply:
ok, thanks for providing samples!

Reply:
first commit is still needed. it makes (at least) english subs work. I can drop the second one for now if you want me to.